releasenotes/notes/remote-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+
+upgradeNotes:
+  - title: Allow opting out of of remote cluster support.
+    content: By setting PILOT_REMOTE_CLUSTER_ACCESS to false it is possible to disable workloads from other clusters from connecting to an Istiod that is not intended as 'primary/remote', increasing the security and isolation.
+  - title: Allow control over cross-cluster secret access.
+    content: PILOT_REMOTE_SECRET_ACCESS prevents Istiod from returning gateway secrets from remote clusters. This is not a common use case and defaults to disalbed - if you use DestinationRules with primary/remote and secrets in different cluster you need to enable the setting. 
+
```

Comment:
Might be good to specify a releaseNotes block here as well with the changes so that it appears in the changelog.

Reply:
Right, will add one - I thought the content above gets included.